tests/FSharp.Test.Utilities/FSharp.Test.Utilities.fsproj

```diff
@@ -16,6 +16,7 @@
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <!--Extra xUnit customizations, not required for the test suite to work, but can speed up local test runs and help with debugging.-->
     <DefineConstants>XUNIT_EXTRAS</DefineConstants>
+    <NoWarn>$(NoWarn);NU1510</NoWarn> <!-- NU1510: Project is explicitly referencing the runtime assembly 'System.Collections.Immutable', however, if we remove it, it tries to find it on the wrong path -->
```

Comment:
Consider setting this directly on the PackageReference so that it only applies to that one:

```<PackageReference Include="..." Version="..." NoWarn="1510" />```